grafanalib/core.py

```diff
@@ -213,6 +213,31 @@ def DashboardLink(dashboard, uri, keepTime=True, title=None):
     }
 
 
+def Template(default, dataSource, label, name, query):
```

Comment:
Actually, after looking at https://github.com/weaveworks/monitoring/pull/113 I have no idea what it means.